backend/store/store.go

```diff
@@ -103,10 +103,14 @@ type SelectionPredicate struct {
 	Limit int64
 	// Subcollection represents a sub-collection of the primary collection
 	Subcollection string
+	// Ordering indicates the property to sort on, if supported by the store
+	Ordering string
+	// OrderDirection indicates the sorting direction
+	OrderingDirection string
```

Comment:
Not to bikeshed too much here but this really only needs to be a boolean.

What about OrderingDirection bool with ASC and DESC constants for true/false, or a field named Descending which defaults to false by way of zero value?

Reply:
I like the bool idea. It probably would be up to the store itself to convert it to whatever format it needs to.

Reply:
Updated.